optuna_integration/catboost/requirements.txt

```
@@ -1 +1,2 @@
 catboost
+numpy<2.0.0
```

Could you add this commen?
```suggestion
numpy<2.0.0  # TODO(nabe): Remove this constraint once this issue is resolved. https://github.com/numpy/numpy/issues/26710
```